postgres-appliance/configure_spilo.py

```diff
@@ -287,6 +290,47 @@ def write_crontab(placeholders, path, overwrite):
     c.communicate(input='\n'.join(lines).encode())
 
 
+def write_ldap_configuration(placeholders, overwrite):
+    if not 'LDAP_URL' in placeholders:
```

Comment:
The idiom is `key not in dict`, I believe ;-)  Also, this could be written as
```
ldap_url = placeholders.get('LDAP_URL')
if ldap_url is NoneL
    # complain and bail out
```